playbooks/appserver.yml

```diff
@@ -8,7 +8,7 @@
     - role: consul
       when: consul_servers | length > 0
     - role: node-exporter
-      when: NODE_EXPORTER_PASSWORD != 'set-me-please'
+      when: NODE_EXPORTER_PASSWORD is not none
```

Comment:
I've filed https://github.com/open-craft/opencraft/pull/312 to change this.

I don't think the integration tests would actually have failed, since the default on the Ocim side still was `set-me-please`.  However, the node exporter would still be installed even if the default isn't changed, and that is certainly undesired.

Reply:
:+1:'d that PR